Launcher/Classes/LocalizationHandler.swift

```diff
@@ -0,0 +1,22 @@
+import Foundation
+
+class LocalizationHandler {
+    
+    let applicationStateHandler = ApplicationStateHandler()
+    
+    func getKey(forString: String) -> [String] {
```

Comment:
If this *is* able to return more than one result, please name it `getKeys`. Also, what about `getKey(for string: String)`?